website_portal_contract/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Website Portal Contract",
+    "summary": "Extends website portal with contracts.",
+    "version": "10.0.1.0.0",
+    "category": "Contract Management",
+    "website": "https://laslabs.com",
+    "author": "LasLabs, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "contract",
+        "website_quote",
```

Comment:
Since both modules have the exact same general xml structure, some of the javascript functionality that already exists in website_quote can be re-used in this module.